doc/docs-revamp/dev-onboarding-quick-setup.md

```diff
@@ -0,0 +1,375 @@
+
+### Redeemer
+A redeemer is a piece of data that is provided by the spending transaction. It is used by the validator script to make decisions during the validation process. The redeemer is used to pass dynamic information to the validator script that may not be known at the time the UTXO is created but is only known at the time the UTXO is being spent. In the context of a vesting contract, the redeemer is typically not used, since the contract's logic does not need any additional input from the transaction. Instead, the validation of the transaction is based solely on the datum (vesting conditions) and the script context, such as checking if the deadline has passed and if the transaction is signed by the beneficiary. However, if required, a redeemer could be used to provide additional arguments to the validator script.
+
+### Script context
+The script context is crucial for a vesting contract as it provides information about the current state of the blockchain and the transaction being validated. The contract's validator script uses the script context to check if the current time has passed the vesting deadline and if the transaction is signed by the beneficiary. This information is necessary to determine whether the conditions for releasing the vested funds have been met.
+
+### Validator scripts
+In a vesting contract, the validator script is responsible for enforcing the vesting conditions and ensuring that the funds are only released to the beneficiary after the specified deadline. The validator script takes the datum (vesting conditions) and the script context as inputs and uses this information to validate the transaction. If the current time is beyond the deadline and the transaction is signed by the beneficiary, the validator script allows the transaction to consume the vested funds. Otherwise, the transaction is deemed invalid, and the funds remain locked in the contract's UTxO.
+
+## Writing your first Plutus smart contract
+
+Now that you have a basic understanding of the core concepts, let's create a simple vesting contract. 
+
+### Setting up a Plutus project
+To set up a Plutus project for our vesting contract, follow these steps:
+
+1. Create a new directory for your project and navigate to it in your terminal.
+
+2. If you are using Nix, enter a Nix shell that provides the necessary development environment by running `nix develop` in your project directory. If you are not using Nix, make sure that all required C libraries are installed since PlutusTx depends on `cardano-base`, which in turn depends on cryptographic C libraries like `libblst`, `libsecp256k1`, and `libsodium`. 
```

Comment:
This part is no longer needed following https://github.com/IntersectMBO/plutus/pull/5864